docker-compose.yml

```
@@ -72,15 +76,15 @@ services:
       options:
           max-file: "5"
           max-size: "5m"
+    labels:
+      - "com.centurylinklabs.watchtower.enable=${AUTO_UPDATE:-true}"
 
   # Watchtower automatically updates the containers
   watchtower:
     image: containrrr/watchtower
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
     command: --interval 3600 --cleanup
```

I thought you were gonna add the `enabled` flag to the `watchtower` Docker Compose service defined in `docker-compose.yml`, but this works too. It's probably a good idea to enable the whitelist command option to watchtower like this, too. 
https://containrrr.dev/watchtower/arguments/#filter_by_enable_label

As long as you tested the `.env` `AUTO_UPDATE` value locally, LGTM

```suggestion
    command: --label-enable --interval 3600 --cleanup
```